spec/spec_helper_acceptance.rb

```diff
@@ -1,11 +1,6 @@
+# This file is completely managed via modulesync
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
-configure_beaker do |host|
-  # The CentOS docker image has a yum config that won't install docs, to keep used space low
```

Comment:
Hey @bastelfreak, I think removing those lines might be the reason why the tests fails.

Reply:
oh right, I can even remember that people debugged that on IRC. the docs packages contain the sql files.